.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -40,3 +40,14 @@ _The author of the PR should fill out the following sections to ensure this PR i
   - [ ] If the amount of work to write unit tests for this change are excessive,
 please explain why (so that we can fix it whenever it gets refactored).
 - [ ] Confirmed relevant documentation has been updated/added.
+
+## Reviewer Checklist 
+_The reviewer of the PR should copy-paste this template into the review comments on review._
+
+- [ ] Linting code actions have passed.
+- [ ] Ran the code locally based on the test instructions. 
+  - [ ] I don't think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
```

Comment:
I do think this should be tested locally - any CSS change could have unintended impacts.

Reply:
ok. I can update. I guess it's a balancing game of time

Reply:
updated @nealmalkaniwapo

Reply:
updated @nealmalkaniwapo

Reply:
Probably a Sara or Luke or your teams call but since code changes here affect so many clients now, I'd say better spend the time in the PRs instead of after a potential bug goes out.

Reply:
hmm. thinking now about the role of QA then.

Reply:
The dev who is making the change, not the reviewer should be responsible for checking for unintended side effects. The review is a second check that everything looks good.